### PR TITLE
pure -> memo

### PR DESCRIPTION
**Pull request by acdlite**

### CODE REVIEW

File: packages/shared/ReactSymbols.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const REACT_FORWARD_REF_TYPE = hasSymbol
 export const REACT_SUSPENSE_TYPE = hasSymbol
   ? Symbol.for('react.suspense')
   : 0xead1;
-export const REACT_PURE_TYPE = hasSymbol ? Symbol.for('react.pure') : 0xead3;
+export const REACT_MEMO_TYPE = hasSymbol ? Symbol.for('react.memo') : 0xead3;
```
Comment (gaearon) [inline packages/shared/ReactSymbols.js]: Yay DevTools

File: packages/shared/getComponentName.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ function getComponentName(type: mixed): string | null {
         return 'Context.Provider';
       case REACT_FORWARD_REF_TYPE:
         return getWrappedName(type, type.render, 'ForwardRef');
-      case REACT_PURE_TYPE:
+      case REACT_MEMO_TYPE:
         return getWrappedName(type, type.render, 'Pure');
```
Comment (gaearon) [inline packages/shared/getComponentName.js]: Should be `'Memo'`

**Comment by acdlite** [comment]
I'll wait until the other `pure`-related PRs land

**Comment by sebmarkbage** [comment]
I think we're ready for a rebase

**Comment by gaearon** [comment]
https://github.com/facebook/react/pull/13905

**Pull request closed without merging**